src/stores/auth.js

```diff
@@ -31,6 +32,9 @@ export const useAuthStore = defineStore('auth', {
       this.authenticated = false;
       this.username = null;
     },
+    setLoginRedirect(redirectPath) {
+      this.loginRedirect = redirectPath;
+    },
```

Comment:
This is a weird setup for a one time use value as you need to reset it on use. It might make more sense to have a method that also does the redirect and clears the value for you, though I still don't think that feels exactly right.